docker/Dockerfile

```diff
@@ -7,11 +7,19 @@ USER naksha
 WORKDIR /home/naksha
 
 # Copy our artifacts into image
-COPY build/libs/naksha-*-all.jar ./app/
+COPY --chown=naksha:root build/libs/naksha-*-all.jar ./app/
```

Comment:
Wanted to understand why chown was required, as it worked fine on my local machine without it.

Reply:
sed command and shell execution was failing because of permission issue, thats why copy commands are adjusted to ensure that the 'naksha' user owns the files, it will avoid unnecessary file permissions issues.

Reply:
alright